src/Imgix/UrlHelper.php

```diff
@@ -10,23 +10,34 @@ class UrlHelper {
     private $signKey;
     private $params;
 
-    public function __construct($domain, $path, $scheme = "http", $signKey = "", $params = array(), $rawEncodePath = false) {
+    public function __construct($domain, $path, $scheme = "http", $signKey = "", $params = array()) {
```

Comment:
I think this changes means you can also delete `$rawEncodePath` from both the signature and the call to this constructor in `UrlBuilder`'s `createURL` method.

Reply:
Good catch, thanks